package.json

```diff
@@ -7,22 +7,32 @@
 	"license": "Unlicense",
 	"devDependencies": {
 		"@openzeppelin/contracts": "2.4.0",
-		"@truffle/contract": "4.0.38",
-		"@truffle/workflow-compile": "2.1.11",
-		"@types/bn.js": "4.11.5",
+		"@peculiar/webcrypto": "1.0.21",
+		"@types/jasmine": "3.4.6",
+		"@types/memdown": "3.0.0",
+		"@types/node": "12.12.7",
+		"@types/node-fetch": "2.5.3",
+		"@zoltu/ethereum-abi-encoder": "3.2.2",
+		"@zoltu/ethereum-crypto": "2.1.1",
+		"@zoltu/ethereum-fetch-json-rpc": "10.1.0",
+		"@zoltu/ethereum-types": "8.2.2",
+		"@zoltu/solidity-typescript-generator": "3.0.2",
 		"ganache-cli": "6.7.0",
+		"jasmine": "3.5.0",
+		"memdown": "5.1.0",
+		"node-fetch": "2.6.0",
 		"solc": "0.5.12",
-		"truffle": "next",
-		"truffle-typings": "1.0.8",
 		"ts-node": "8.4.1",
-		"typechain": "1.0.3",
-		"typechain-target-truffle": "1.0.0",
-		"typescript": "3.6.4"
+		"typescript": "3.7.2"
+	},
+	"dependencies": {
+		"ganache-core": "2.8.0"
 	},
-	"dependencies": {},
 	"scripts": {
-		"start-ganache": "ganache-cli --gasLimit 9600000",
-		"build": "truffle compile && typechain --target truffle ./build/**/*.json",
-		"deploy": "truffle migrate --reset"
+		"compile": "npx ts-node ./scripts/compile/index.ts",
```

Comment:
I believe npx here is redundant, since all scripts execute in the environment of the npm